example-common/repository-jpa/src/main/java/org/apache/shardingsphere/example/common/jpa/service/JPACommonServiceImpl.java

```diff
@@ -36,6 +39,9 @@
     
     @Resource
     private OrderItemRepository orderItemRepository;
+
+    @Resource
+    private CountryRepository countryRepository;
```

Comment:
you can add a CountryServiceImpl to isolate the modification like we have done in UserServiceImpl